src/app/AppProvider.tsx

```
@@ -0,0 +1,20 @@
+import { ReactNode, useLayoutEffect } from "react";
+
+export default function AppProvider({
+  children,
+  initialSessionToken = "",
+}: {
+  children: ReactNode;
+  initialSessionToken?: string;
+}) {
+  useLayoutEffect(() => {
+    // MEMO: Only reassigning value in client component
+    if (typeof window !== undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)